api/firmware/attestation.go

```diff
@@ -78,23 +79,33 @@ func (device *Device) performAttestation() (bool, error) {
 	if err != nil {
 		return false, err
```

Comment:
Don't you want to log the error here as well?

Reply:
Here the error is returned and logged in the callsite.

Reply:
added log anyway